orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
> That's just a warning about using hypothesis in python manage.py shell I think. The standard usage in tests is well supported.

That's good at least.  There are unfortunately lots more caveats though.  (I was looking at https://hypothesis.readthedocs.org/en/latest/django.html)